lib/gems/pending/util/miq-password.rb

```diff
@@ -72,6 +73,28 @@ def self.encrypted?(str)
     !!split(str).first
   end
 
+  def self.encrypt_hash(values, &block)
```

Comment:
Please add comments on what do you expect from the block - the purpose and the return type.